3term-study/행렬의-덧셈.js

```diff
@@ -0,0 +1,3 @@
+function solution(arr1, arr2) {
+  return arr1.map((a, row) => a.map((b, col) => b + arr2[row][col]));
```

Comment:
```suggestion
  return arr1.map((elArr, row) => elArr.map((el, col) => el + arr2[row][col]));
```
콜백함수의 매개변수 작명도 어느정도 의미부여를 해주세요 (저도 작명 센스가 없어서 일단 예시로 했습니다)